tests/test_notebooks.py

```diff
@@ -55,6 +55,14 @@ def test_multibinpois(common_kwargs):
     assert nb.scraps['number_2d_successpoints'].data > 200
 
 
+def test_pullplot(common_kwargs):
+    pm.execute_notebook('docs/examples/notebooks/pullplot.ipynb', **common_kwargs)
```

Comment:
Would also kinda be nice to have some consistent naming convention between notebooks. But that's something for another PR.